<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RVSettingsDialog</class>
 <widget class="QDialog" name="RVSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RV Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Launch using</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="uiExecDDL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>RV</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RVSDI</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Path Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="uiNetworkPathsCHK">
          <property name="text">
           <string>Use Network Paths</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="uiLocalPathsCHK">
          <property name="text">
           <string>Use Local Paths</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Local prefix</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="uiLocalPrefixTXT">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="uiLocalPrefixBTN">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="uiConvertToDpxCHK">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Convert to DPX filepaths</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Additional Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="uiProfileDDL">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="uiSubprocessCHK">
        <property name="text">
         <string>Run in separate process</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="uiDialogBTNS">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>uiExecDDL</tabstop>
  <tabstop>uiNetworkPathsCHK</tabstop>
  <tabstop>uiLocalPathsCHK</tabstop>
  <tabstop>uiLocalPrefixTXT</tabstop>
  <tabstop>uiProfileDDL</tabstop>
  <tabstop>uiSubprocessCHK</tabstop>
  <tabstop>uiDialogBTNS</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>uiDialogBTNS</sender>
   <signal>accepted()</signal>
   <receiver>RVSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiDialogBTNS</sender>
   <signal>rejected()</signal>
   <receiver>RVSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiLocalPathsCHK</sender>
   <signal>toggled(bool)</signal>
   <receiver>uiLocalPrefixTXT</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiLocalPathsCHK</sender>
   <signal>toggled(bool)</signal>
   <receiver>uiConvertToDpxCHK</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
